rasterfoundry/models/project.py

```diff
@@ -192,7 +191,15 @@ def get_image_source_uris(self):
         source_uris = []
         scenes = self.api.client.Imagery.get_projects_uuid_scenes(uuid=self.id) \
                      .result().results
+        scene_order = self.api.client.Imagery.get_projects_uuid_order(uuid=self.id) \
```

Comment:
I think this needs to check whether or not there is a "next" or not -- in cases where the number of scenes is greater than pagination I think this will return an incomplete list of UUIDs. Can probably use a `while` loop to make sure all the IDs are included in the list.